engine/execution/computation/manager.go

```diff
@@ -240,7 +240,6 @@ func DefaultFVMOptions(chainID flow.ChainID, cadenceTracing bool, extensiveTraci
 					AttachmentsEnabled: chainID != flow.Mainnet,
 				},
 			)),
-		fvm.WithEVMEnabled(true),
```

Comment:
Looking at the option above, does that mean that attachments are currently broken on VNs too?

Reply:
ah, good catch.... yes.

let me address that.